tests/test_adjoint_jacobian.py

```diff
@@ -997,7 +1088,12 @@ def test_interface_jax(self, dev):
         jax interface"""
 
         jax = pytest.importorskip("jax")
-        if dev.R_DTYPE == np.float64:
+        if dev._new_API:
+            dtype = np.float64
+        else:
+            dtype = dev.R_DTYPE
+
+        if dtype == np.float64:
```

Comment:
Same as above with this jax test. Only `np.float64` works with the new device regardless of `dev.c_dtype`.

Reply:
Could you add here a link to the error message?

Reply:
I only got that failure locally and updated the test to its current hacky state. See below comment for more details.

Reply:
JAX failure was local due to incorrect environment variables being used